ide/coqide/coqide.ml

```diff
@@ -340,10 +340,10 @@ let check_quit ?parent saveall =
    with e -> flash_info ("Cannot save preferences (" ^ Printexc.to_string e ^ ")"));
   let is_modified sn = sn.buffer#modified in
   if List.exists is_modified notebook#pages then begin
-    let answ = Configwin_ihm.question_box ~title:"Quit"
-      ~buttons:["Save Named Buffers and Quit";
-                "Quit without Saving";
-                "Don't Quit"]
+    let answ = Ui_dialogs.question_box ~title:"Quit"
+      ~buttons:[ButtonWithLabel "Save named buffers and quit";
+                ButtonWithLabel "Quit without saving";
+                ButtonWithStock `CANCEL]
```

Comment:
```suggestion
                ButtonWithLabel "Don't Quit"]
```

I think the original button text is clearer.

Reply:
OK, restored.